src/pages/patch/buildVariants/GroupedTaskSquare.tsx

```diff
@@ -12,20 +12,23 @@ interface Props {
   statuses: string[];
   color: string;
   textColor: string;
+  variant: string;
 }
 
 export const GroupedTaskSquare: React.FC<Props> = ({
   statuses,
   count,
   color,
   textColor,
+  variant,
 }) => {
   const patchAnalytics = usePatchAnalytics();
   const { id } = useParams<{ id: string }>();
 
   const nextQueryParams = queryString.stringify(
     {
       statuses,
+      variant,
```

Comment:
i think we should expand and overwrite query params in case the patch page relies on additional query params in the future